### PR TITLE
Remove active state from selection property

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Screenshot 2024-04-24 at 13 58 09

## What is the new behavior?
Screenshot 2024-04-24 at 13 57 53

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/symbols/Node.tsx [hunk 1]
```diff
@@ -275,7 +275,7 @@ export const Node: FC<NodeProps> = ({
               color={color}
               node={node}
               active={combinedActiveState}
-              selected={isSelected || isActive}
+              selected={isSelected}
```
Comment (ghsteff) [inline src/symbols/Node.tsx:278]: You could remove this `selected` prop entirely, `Icon` doesn't use it

**Review by ghsteff — APPROVED** [review]
Just confirming, is the intention is to remove the ring from the nodes that aren't selected?

**Comment by amcdnl** [comment]
> Just confirming, is the intention is to remove the ring from the nodes that aren't selected?

Yes.

File: src/symbols/Node.tsx (hunk 1 above)
Comment (amcdnl) [reply src/symbols/Node.tsx:278]: @evgenoid ^^

Comment (evgenoid) [reply src/symbols/Node.tsx:278]: Yes, but it extends `NodeRendererProps` where `selected` is required property

**Pull request merged**